Mathlib/Util/AssertExists.lean

```diff
@@ -8,16 +8,16 @@ import Lean.Elab.Command
 import Mathlib.Util.AssertExistsExt
 
 /-!
-# User commands for assert the (non-)existence of declaration or instances.
+# User commands to assert the (non-)existence of declarations or instances.
 
 These commands are used to enforce the independence of different parts of mathlib.
 
 ## TODO
 
-Potentially after the port reimplement the mathlib 3 linters to check that declarations asserted
-about do eventually exist.
+After the port, potentially reimplement the mathlib 3 linters to check that declarations asserted
```

Comment:
The port has happened, so at the very least we could change this to `Potentially repimplement...`?

Reply:
Fair point. While I was at it, I changed a porting note to a TODO that was also talking about things to do after the port.